AlexeyShparun/hometask-2/get_middle_charachter.rb

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+def get_middle(str)
+  if str.length.even?
+    str[str.length / 2 - 1] + str[str.length / 2]
```

Comment:
`str.length / 2` is repeated 3 times

Consider making a variable for this value